Meta/CMake/link_libc_headers.cmake

```diff
@@ -0,0 +1,12 @@
+cmake_minimum_required(VERSION 3.22)
+
+if (NOT DEFINED SERENITY_ARCH OR NOT DEFINED SERENITY_SYSROOT)
+    message(FATAL_ERROR "SERENITY_ARCH and SERENITY_SYSROOT must be defined")
+endif()
+
+set(CMAKE_CURRENT_SOURCE_DIR "${CMAKE_CURRENT_SOURCE_DIR}/Userland/Libraries/LibC")
```

Comment:
😱

Ha ha. Nice joke, trying to change this internal CMake variable. You shouldn't scare me so much in your next PR with something this cursed.